api-server/stack-test-suite/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
```

Comment:
Why do we need a whole new crate for these tests? The whole point of having in-memory storage is to make it casual... just like we're doing for the chainstate test suite or block production. We can construct a whole chain in unit tests.

Reply:
Since these tests combined the scanner, API storage, and the web server, it didn't make sense to me at the time to have these tests just as unit tests within `web-server`.

> just like we're doing for the chainstate test suite or block production

I copied how `storage-test-suite` and chainstate's `test-framework` was done which also have a separate `[package]`.

Given the reasoning above, I'm happy to move them into the web-server unit tests if you want.

Reply:
That's fine. I just think it's an overkill.